### PR TITLE
Created the event data for London Codurance

### PR DESCRIPTION
**Pull request by MiyamotoAkira**

### CODE REVIEW

File: _data/events_gdcr2019/London_Codurance.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "title": "GDCR19 in London (@Codurance)",
+  "url": "https://www.meetup.com/london-software-craftsmanship/events/265499324/",
```
Comment (rachelcarmena) [inline _data/events_gdcr2019/London_Codurance.json]: The link doesn't work :cry:

Comment (rachelcarmena) [reply _data/events_gdcr2019/London_Codurance.json]: Please, check it here: https://deploy-preview-557--coderetreat-org.netlify.com/events/

File: _data/events_gdcr2019/London_Codurance.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      },
+      {
+          "name": "Jorge Gueorguiev Garcia",
+          "url": "https://twitter.com/yefoakira"
+      }
+  ],
+  "sponsors": [
+    {
+      "name": "Codurance",
+      "url": "codurance.com"
```
Comment (rachelcarmena) [inline _data/events_gdcr2019/London_Codurance.json]: Please, add the url protocol because now... :bomb: :sweat_smile:

Comment (MiyamotoAkira) [reply _data/events_gdcr2019/London_Codurance.json]: Ah, a test that needs to be added? ;-)

Corrected now

File: _data/events_gdcr2019/London_Codurance.json (hunk 1 above)
Comment (MiyamotoAkira) [reply _data/events_gdcr2019/London_Codurance.json]: Oh, I have the entry in preview mode. I was waiting for this to be merged before making it public. It should be visible now

**Comment by rachelcarmena** [comment]
Thanks @MiyamotoAkira and welcome :tada:

**Pull request merged**